api/asset.ts

```diff
@@ -43,3 +45,55 @@ export async function fetchAssets(): Promise<Asset[]> {
   cache = result
   return result
 }
+
+export async function fetchAssetStatuses(): Promise<AssetStatus[]> {
+  const { assetStatuses } = await getAssetStatuses()
+
+  return assetStatuses.map((assetStatus) => {
+    const underlying = toCurrency(assetStatus.asset.underlying)
+    const epoch = {
+      id: +assetStatus.epoch.id,
+      startTimestamp: +assetStatus.market.epoch.startTimestamp,
+      endTimestamp: +assetStatus.market.epoch.endTimestamp,
+    }
+    const market = Market.fromDto({
+      address: getAddress(assetStatus.market.id),
+      orderToken: getAddress(assetStatus.market.orderToken),
+      takerFee: assetStatus.market.takerFee,
+      quoteUnit: assetStatus.market.quoteUnit,
+      epoch: {
+        id: assetStatus.market.epoch.id,
+        startTimestamp: assetStatus.market.epoch.startTimestamp,
+        endTimestamp: assetStatus.market.epoch.endTimestamp,
+      },
+      quoteToken: {
+        address: getAddress(assetStatus.market.quoteToken.id),
+        name: assetStatus.market.quoteToken.name,
+        symbol: assetStatus.market.quoteToken.symbol,
+        decimals: assetStatus.market.quoteToken.decimals,
+      },
+      baseToken: {
+        address: getAddress(assetStatus.market.baseToken.id),
+        name: assetStatus.market.baseToken.name,
+        symbol: assetStatus.market.baseToken.symbol,
+        decimals: assetStatus.market.baseToken.decimals,
+      },
+      depths: assetStatus.market.depths.map((depth) => ({
+        price: depth.price,
+        rawAmount: depth.rawAmount,
+        isBid: depth.isBid,
+      })),
+    })
+    const decimals = assetStatus.asset.underlying.decimals
+    const totalAvailable = formatUnits(market.totalBidsInBase(), decimals)
+    const totalDeposits = formatUnits(assetStatus.totalDeposits, decimals)
```

Comment:
Current `totalDeposits` is only asset-epoch.
I think `totalDeposits` will be accumlate value from 1~N